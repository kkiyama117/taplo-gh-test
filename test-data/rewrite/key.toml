rewrite_me = 2

[table]
rewrite_me = 2

[table.subtable.rewrite_me]
val = 2

[[arr.rewrite_me]]
[[arr.rewrite_me]]
rewrite_me = "rewrite_me"
[[arr.rewrite_me]]
