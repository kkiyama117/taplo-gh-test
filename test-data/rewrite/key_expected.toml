rewritten = 2

[table]
rewritten = 2

[table.subtable.rewritten]
val = 2

[[arr.rewritten]]
[[arr.rewritten]]
rewritten = "rewrite_me"
[[arr.rewritten]]
